ann_benchmarks/main.py

```diff
@@ -70,7 +70,7 @@ def run_worker(cpu: int, args: argparse.Namespace, queue: multiprocessing.Queue)
             run(definition, args.dataset, args.count, args.runs, args.batch)
         else:
             memory_margin = 500e6  # reserve some extra memory for misc stuff
-            mem_limit = int((psutil.virtual_memory().available - memory_margin) / args.parallelism)
+            mem_limit = int((psutil.virtual_memory().total - memory_margin) / args.parallelism)
```

Comment:
Please remove this

Reply:
Hi @maumueller, this change is related to #547. And now we just modified it with the suggested way in the issue. Please have a check.

Reply:
Thanks for modifying it in the suggested way.